pyworklistserver/worklist_server.py

```diff
@@ -1,40 +1,66 @@
 """ DICOM server implementation that responds to C-FIND requests"""
+import random
+import os
+import time
 from pynetdicom import AE
 from pynetdicom import evt
 from pynetdicom.sop_class import ModalityWorklistInformationFind
 
 from pyworklistserver import worklist
+from pyworklistserver import user_config
+from pyworklistserver import get_config
 
 class WorklistServer:
     """ Simple worklist server with support for one client. This
     implementation class is responsible for network communication with
     the client. """
 
-    def __init__(self, serverConfig, app_logger, seed, blocking):
+    def __init__(self, serverConfig, app_logger, blocking):
         self._config = serverConfig
+        self._config_values = get_config.GetConfig()
         self._logger = app_logger
         self._blocking = blocking
-        self._seed = seed
-        self._worklist_factory = worklist.RandomWorklist('ISO_IR 100', self._seed)
+        self._worklist_factory = worklist.RandomWorklist('ISO_IR 100', self._config_values)
         self._handlers = [
             (evt.EVT_C_FIND, self._on_find, [app_logger]),
         ]
         self._server = None
 
+    def get_seedNumber(self):
+        """Return Function for seed-value"""
+        if self._config_values.seed_Number == 0:
+            return random.randrange(1, 1000000)
+        return self._config_values.seed_Number
+
+    def setup_log_seed(self):
+        if os.path.exists("logfile.txt"):
+            os.remove("logfile.txt")
+        f = open("logfile.txt", "w+")
+        f.write('User_config:\trateOfRandomExams: %d\t rateOfCleanExams: %d\t minAmountOfWorklistExams: %d\t maxAmountOfWorklistExams: %d\t seed_Number (set): %d\n likelyhood_of_long_string: %d\t likelyhood_of_empty_string: %d\t likelyhood_of_None_string: %d\t likelyhood_of_delay: %d\t is_long: %d\t is_empty: %d\t is_None: %d\t is_delay: %d \n\n'
+        % (user_config.rateOfRandomExams, user_config.rateOfCleanExams, user_config.minAmountOfWorklistExams, user_config.maxAmountOfWorklistExams, user_config.seed_Number, user_config.likelyhood_of_long_string, user_config.likelyhood_of_empty_string, user_config.likelyhood_of_None_string, user_config.likelyhood_of_delay, user_config.long_enabled, user_config.empty_enabled, user_config.none_enabled, user_config.delay_enabled))
+
+        f.close()
+
+    def log_seed (self, seed):

```

Comment:
I think what you are referring to is writing a config file, not logging. Please rename to better reflect this.